src/transformers/models/vit_mae/modeling_vit_mae.py

```diff
@@ -213,21 +213,6 @@ def __init__(self, config):
             torch.zeros(1, self.num_patches + 1, config.hidden_size), requires_grad=False
         )
         self.config = config
```

Comment:
I think you might need the `post_init()` call here

```suggestion
        self.config = config
        
        # Initialize weights and apply final processing
        self.post_init()
```